conjure-api/build.gradle

```diff
@@ -26,9 +26,7 @@ project (':conjure-api:conjure-api-objects') {
 }
 
 project (':conjure-api:conjure-api-typescript') {
-    publishTypeScript.doFirst {
-        file('src/.npmrc') << "//registry.npmjs.org/:_authToken=${System.env.NPM_AUTH_TOKEN}"
-    }
+    generateNpmrc.token = System.env.NPM_AUTH_TOKEN;
```

Comment:
Do we need to update this as well? https://github.com/palantir/conjure/blob/731f08dacd665410b6fb6ed00fd29ae477d63fc0/docs/howto/publish_typescript_to_npm.md

Reply:
Yes we do